settings.gradle

```diff
@@ -1 +1,3 @@
 rootProject.name = 'tw-base-utils'
+
+include 'tw-base-utils'
```

Comment:
I moved the sources to a module, so it will look the same other libs and I can just copy paste all the build files.